components/automate-ui/src/app/pages/data-feed-create/data-feed-create.component.ts

```diff
@@ -203,6 +203,13 @@ export class DataFeedCreateComponent {
         this.integrationSelected = true;
         break;
       }
+      case WebhookIntegrationTypes.ELK_KIBANA: {
+        this.showFieldWebhook();
+        this.authSelected = AuthTypes.USERNAMEANDPASSWORD;
```

Comment:
In story ELK page open with default authentication as Access token, is this ok to show with Username and password?